flatgfa/src/main.rs

```diff
@@ -0,0 +1,32 @@
+use std::env;
+
+use flatgfa::{
+    memfile::{self, *},
+    packedseq::{self, *},
+};
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() > 2 && args[1] == "import" {
+        let mmap = memfile::map_file(&args[2]); // args[2] is the filename
+        let seq = packedseq::view(&mmap);
+        println!("Sequence: {}", seq);
```

Comment:
I say let's just print the raw `seq` itself, without the `Sequence: ` prefix. This will make the CLI friendlier for use in larger pipelines.